google-cloud-firestore/src/main/java/com/google/cloud/firestore/Query.java

```diff
@@ -1638,11 +1645,22 @@ private void internalStream(
       request.setReadTime(readTime.toProto());
     }
 
+    traceUtil
+        .currentSpan()
+        .addEvent(
+            TraceUtil.SPAN_NAME_RUN_QUERY,
+            new ImmutableMap.Builder<String, Object>()
+                .put("transactional", transactionId != null)
```

Comment:
nit - for consistency, maybe `isTransactional` or remove the `is` from `retryRequestWithCursor`

Reply:
Agreed. Updated to `isTransactional`..